.github/workflows/conformance-e2e.yaml

```diff
@@ -1,474 +0,0 @@
-        required: false
-        default: '{}'
-  push:
-    branches:
-      - 'renovate/v1.15-**'
-
-# By specifying the access of one of the scopes, all of those that are not
-# specified are set to 'none'.
-permissions:
-  # To be able to access the repository with actions/checkout
-  contents: read
-  # To allow retrieving information from the PR API
-  pull-requests: read
-  # To be able to set commit status
-  statuses: write
-
-concurrency:
-  # Structure:
-  # - Workflow name
-  # - Event type
-  # - A unique identifier depending on event type:
-  #   - schedule: SHA
-  #   - workflow_dispatch: PR number
-  #
-  # This structure ensures a unique concurrency group name is generated for each
-  # type of testing, such that re-runs will cancel the previous run.
-  group: |
-    ${{ github.workflow }}
-    ${{ github.event_name }}
-    ${{
-      (github.event_name == 'push' && github.sha) ||
-      (github.event_name == 'schedule' && github.sha) ||
-      (github.event_name == 'workflow_dispatch' && github.event.inputs.PR-number)
-    }}
-  cancel-in-progress: true
-
-env:
-  cilium_cli_ci_version:
-
-jobs:
-  echo-inputs:
-    if: ${{ github.event_name == 'workflow_dispatch' }}
-    name: Echo Workflow Dispatch Inputs
-    runs-on: ubuntu-24.04
-    steps:
-      - name: Echo Workflow Dispatch Inputs
-        run: |
-          echo '${{ tojson(inputs) }}'
-
-  commit-status-start:
-    name: Commit Status Start
-    runs-on: ubuntu-latest
-    steps:
-      - name: Set initial commit status
-        uses: myrotvorets/set-commit-status-action@3730c0a348a2ace3c110851bed53331bc6406e9f # v2.0.1
-        with:
-          sha: ${{ inputs.SHA || github.sha }}
-
-  wait-for-images:
-    name: Wait for images
-    runs-on: ubuntu-latest
-    timeout-minutes: 30
-    steps:
-      - name: Checkout context ref (trusted)
-        uses: actions/checkout@d632683dd7b4114ad314bca15554477dd762a938 # v4.2.0
-        with:
-          ref: ${{ inputs.context-ref || github.sha }}
-          persist-credentials: false
-      - name: Wait for images
-        uses: ./.github/actions/wait-for-images
-        with:
-          SHA: ${{ inputs.SHA }}
-
-  setup-and-test:
-    needs: [wait-for-images]
-    runs-on: ${{ vars.GH_RUNNER_EXTRA_POWER }}
-    name: 'Setup & Test'
-    env:
-      job_name: 'Setup & Test'
-    strategy:
-      fail-fast: false
-      max-parallel: 16
-      matrix:
-        include:
-          # !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
-          # ! NOTE: keep tests-e2e-upgrade.yaml config in sync !
-          # !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
-
-          - name: '1'
-            # renovate: datasource=docker depName=quay.io/lvh-images/kind
-            kernel: '4.19-20240710.064909'
-            kube-proxy: 'iptables'
-            kpr: 'false'
-            tunnel: 'vxlan'
-            host-fw: 'true'
```

Comment:
To note - in the `tests-e2e-upgrade` workflow the `host-fw` option is set to false (to work-around downgrade bugs). So if we now solely use that workflow for testing, we're effectively losing coverage.

Reply:
@julianwiedmann thanks for the review. I've copied the host-fw's specific tests intot he tests-e2e-upgrade tests and skip the upgrade for those configurations. We shouldn't be losing any coverage now.